### PR TITLE
[READY] Issue #1626 - hs.tangent - Add support for Tangent Panels & iPad App

### PR DESCRIPTION
**Pull request by latenitefilms**
As discussed in #1626.

### CODE REVIEW

**Comment by cmsj** [comment]
@latenitefilms since I'm not really familiar with these devices, I'm happy to defer to you on when this is ready to merge :)

**Comment by latenitefilms** [comment]
@cmsj - No problemo, will keep you posted! I just wanted to get it into a PR sooner rather than later so that I can get help earlier when I run into roadblocks.

File: extensions/tangent/init.lua [hunk 1]
```diff
@@ -0,0 +1,1499 @@
...
+--  * signed - `true` if it's a signed integer otherwise `false`
+--
+-- Returns:
+--  * A number value
+local function byteStringToNumber(str, offset, numberOfBytes, signed)
+	local format = ">I" .. tostring(numberOfBytes)
+	if signed then
+		format = ">i" .. tostring(numberOfBytes)
+	end
+  return string.unpack(format, str:sub(offset, offset + numberOfBytes - 1))
```
[the remark this answers is not in the document]
Comment (latenitefilms) [reply extensions/tangent/init.lua]: @cmsj - Does this mean @stickler-ci is running an old version of Lua?

**Comment by latenitefilms** [comment]
@cmsj & @asmagill - I think this is pretty good to go now, however if you want to have a quick review of the code to make sure I'm not doing anything stupid, that would be great!

**Comment by latenitefilms** [comment]
@asmagill & @cmsj - This code has been in the public CommandPost beta for a few weeks now and all seems to be working fine, so feel free to merge away!

**Pull request merged**